src/staging/gatsby-node.js

```diff
@@ -263,6 +274,37 @@ async function createParks({ graphql, actions, reporter }) {
   })
 }
 
+async function createSites({ graphql, actions, reporter }) {
+  const siteQuery = `
+  {
+    allStrapiSites {
+      nodes {
+        id
+        siteName
+        siteNumber
+        orcsSiteNumber
+        protectedArea {
+          urlPath
+        }
+      }
+      totalCount
+    }
+  }
+  `
+  const result = await strapiApiRequest(graphql, siteQuery)
+
+  result.data.allStrapiSites.nodes.forEach(site => {
+    const slug = slugify(site.siteName).toLowerCase()
```

Comment:
Do we follow this pattern elsewhere? I don't like it, it feels like generally on this site we set the slug manually, and I'd like to continue with that patten here for consistency.

So instead of using the page title, I would prefer to add a `siteUrl` field that could be set manually.

If the issue is that we can't grab that with an import, perhaps we could add the new field, and use the current scheme as a fallback.

Feel free to discuss offline if there's a reason I can't see.

Reply:
This is because a new field won't be available on Gatsby when this PR will be merged.
To avoid the build error, I can create and include a new field `siteUrl` or `slug` in this commit but applying the new field to this site path logic will be after this ticket deployment.